cli/js/buffer.ts

```diff
@@ -39,7 +43,8 @@ export class Buffer implements Reader, ReaderSync, Writer, WriterSync {
     this.#buf = new Uint8Array(ab);
   }
 
-  bytes(): Uint8Array {
+  bytes(options: BytesOptions = { copy: true }): Uint8Array {
```

Comment:
Maybe this can be done without introduce the new interface?

```ts
bytes(options: { copy: boolean } = { copy: true }): UintArray
```

Reply:
I had it that way initially, then I added the interface, I'll change it back.